src/applications/vaos/services/appointment/index.js

```diff
@@ -360,10 +263,10 @@ export function getVideoAppointmentLocation(appointment) {
  */
 export function getVAAppointmentLocationId(appointment) {
   if (
-    isVideoAppointment(appointment) &&
-    appointment.vaos.appointmentType === APPOINTMENT_TYPES.vaAppointment
+    appointment?.vaos.isVideo &&
+    appointment?.vaos.appointmentType === APPOINTMENT_TYPES.vaAppointment
   ) {
-    return getVideoAppointmentLocation(appointment);
+    return appointment.videoData.facilityId;
```

Comment:
Eventually, I think I want to rollup the location/facility id to the top level of the appointment object for all types, so we don't need this helper at all.